test/ir_Airwell_test.cpp

```diff
@@ -250,4 +251,142 @@ TEST(TestDecodeAirwell, RealExample3) {
   EXPECT_EQ(0x60080002, irsend.capture.value);
   EXPECT_EQ(0x0, irsend.capture.address);
   EXPECT_EQ(0x0, irsend.capture.command);
+  EXPECT_EQ(
```

Comment:
This is the only testdata that is valid for this AC? If not should it be added to the above as well.

And if the other data also is an ac, should there be models added? (even if not I just want to make a mental note about it)

Reply:
> This is the only testdata that is valid for this AC? If not should it be added to the above as well.

No. Yes, added/fixed. Yeah, I was lazy.

> And if the other data also is an ac, should there be models added? (even if not I just want to make a mental note about it)

It appears it's all a single common "protocol" based on comparing the comments in the Original issue #1069 describing the raw captures and what the library decodes them to.